website/docs/d/is_security_group_rule.html.markdown

```diff
@@ -0,0 +1,61 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_is_security_group_rule"
+description: |-
+  Get information about is_security_group_rule
+subcategory: "Virtual Private Cloud API"
+---
+
+# ibm_is_security_group_rule
+
+Provides a read-only data source for is_security_group_rule. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
+
+## Example Usage
+
+```hcl
+data "ibm_is_security_group_rule" "example" {
+    security_group_rule = ibm_is_security_group_rule.example.rule_id
+    security_group = ibm_is_security_group.example.id
+}
+```
+
+## Argument Reference
+
+Review the argument reference that you can specify for your data source.
+
+- `security_group_rule` - (Required, Forces new resource, String) The rule identifier.
```

Comment:
Mostly we don't specify Forces new resource.. please check the existing datasources..

Reply:
done.